src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -802,6 +802,39 @@ class MessageHelperTest : public ::testing::Test {
     obj.state = state;
   }
 
+  void CheckMessageParams(mobile_apis::SoftButtonType::eType button_type) {
+    smart_objects::SmartObject soft_button;
+    soft_button[strings::system_action] =
+        mobile_apis::SystemAction::INVALID_ENUM;
+    soft_button[strings::type] = button_type;
+
+    smart_objects::SmartObject message_params;
+    smart_objects::SmartObject& msg_param =
+        message_params[strings::soft_buttons][0];
+    msg_param = soft_button;
```

Comment:
@indlu please remove couple of redundant variables that make code more complicated 
```
smart_objects::SmartObject soft_button = message_params[strings::soft_buttons][0];
soft_button[strings::system_action] =
+        mobile_apis::SystemAction::INVALID_ENUM; 
...
soft_button[strings::text] = button_text;
```